float64[] arm_dim
float64[] arr_jointPos
float64[] xyz_coordinates
---
float64[] final_value